test/parallel/test-assert.js

```diff
@@ -330,10 +331,13 @@ assert.throws(function() {assert.ifError(new Error('test error'));});
 assert.doesNotThrow(function() {assert.ifError(null);});
 assert.doesNotThrow(function() {assert.ifError();});
 
-assert.throws(() => {
-  assert.doesNotThrow(makeBlock(thrower, Error), 'user message');
-}, /Got unwanted exception. user message/,
-   'a.doesNotThrow ignores user message');
+assert.throws(
+  () => {
```

Comment:
By the way, the previous indentation is, in my opinion at least, unfriendly to the reader:

```javascript

assert.throws(() => {
  assert.doesNotThrow(makeBlock(thrower, Error), 'user message');
}, /Got unwanted exception. user message/,
   'a.doesNotThrow ignores user message');
```

I think moving the arrow function down one line is worth getting code that is easier to understand at a glance:

```javascript
assert.throws(
  () => {
    assert.doesNotThrow(makeBlock(thrower, Error), 'user message');
  },
  /Got unwanted exception. user message/,
  'a.doesNotThrow ignores user message'
);
```